modules/cnf-configure_for_irq_dynamic_load_balancing.adoc

```diff
@@ -24,6 +24,11 @@ spec:
     reserved: 0-1
 ...
 ----
++
```

Comment:
This `+` does _not_ render in the output, and looks the same as the ones that do to me in the source :thinking: Not sure what's up with that!

Reply:
This one is in an ordered list.